app/react-native/package.json

```diff
@@ -56,21 +45,32 @@
     "babel-preset-stage-0": "^6.24.1",
     "babel-runtime": "^6.23.0",
     "case-sensitive-paths-webpack-plugin": "^2.0.0",
-    "commander": "^2.9.0",
     "css-loader": "^0.28.0",
-    "events": "^1.1.1",
-    "express": "^4.15.2",
     "file-loader": "^0.11.1",
     "json-loader": "^0.5.4",
-    "json5": "^0.5.1",
     "postcss-loader": "^2.0.3",
-    "shelljs": "^0.7.7",
+    "react-dom": "^15.5.4",
```

Comment:
These should not be dependencies, as I understand it, so that your app can determine the proper version? @ndelangen ?